website/cue/reference.cue

```diff
@@ -52,7 +52,7 @@ _values: {
 // * `removed` - The component has been removed.
 #DevelopmentStatus: "beta" | "stable" | "deprecated" | "removed"
 
-#EncodingCodec: "json" | "logfmt" | "ndjson" | "text"
+#EncodingCodec: "json" | "logfmt" | "ndjson" | "text" | "native" | "native_json"
```

Comment:
I think you're looking for this place:

https://github.com/vectordotdev/vector/blob/b9ffca3affb927fdb75c8f384c659db0b50679fc/website/cue/reference/components/sources.cue#L157-L176

😄